src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,48 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp(){
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("샛 크기")
```

Comment:
DisplayName에는 다음과 같은 항목이 들어가야 해요
- 테스트 대상
- 테스트 조건
- 예상 결과

```suggestion
    @DisplayName("size 메서드는 가진 아이템의 갯수를 반환한다.")
```
여기서는 클래스명이 어떤 클래스인지를 보여주고 있으니 메서드와 결과를 작성할 수 있겠죠. 
setUp에서 when까지 수행해버렸기에 여기서 조건을 작성할순 없으니 빠질 수도 있구요. 

이처럼 DisplayName혹은 주석등은 메서드명과같이 최대한 시그니처를 간결하게 보여주는게 목적입니다. ㅎㅎ